### ISSUE TITLE
When trying to install got assignment count mismatch error

### ISSUE THREAD

**Issue by FlyersWeb** [jgranstrom/gonodepkg#2]
Hi, I tried to install gonodepkg but I get an issue.

When trying to install using go get, I've got assignment count mismatch: 2 = 3 error : 

```
$ go get github.com/jgranstrom/gonodepkg                                    
# github.com/jgranstrom/gonodepkg
../.go/src/github.com/jgranstrom/gonodepkg/gonodepkg.go:93: assignment count mismatch: 2 = 3
```

**Comment by taytayevanson** [comment]
+1

**Comment by EmirAzaiez** [comment]
+1 I just got the same problem, can't fix it.

**Comment by jgranstrom** [comment]
Fixed with af29c3d9abcdcb18e8c6e64201e36d0a46999fa0
